GTG/gtk/browser/tag_editor.py

```diff
@@ -454,8 +443,7 @@ def on_tn_entry_changed(self, widget):
             # Also, wait 1 second before commiting the change in order to
             # reduce rename requests
             tn_entry_changes = self.watch_tn_entry_changes
-            self.tn_entry_watch_id = GObject.timeout_add(1000,
-                                                         tn_entry_changes)
+            self.tn_entry_watch_id = GObject.timeout_add(1000, tn_entry_changes)
```

Comment:
If it doesn't have to be exact, then `GObject.timeout_add_seconds(1, ...)` would be more appropriate. Just a random note, as this change in the commit isn't about that at all, but just reformatting.